src/content/events/2023/training-sept-2023/wgc.md

```diff
@@ -41,30 +41,42 @@ The training material will be presented virtually by Seqera Labs bioinformatics
             <tbody>
             <tr>
                 <td >9:30 AM</td>
-                <td background-color:navy; rowspan="1">Session 1</td>
+                <td background-color:navy; rowspan="1">
```

Comment:
```suggestion
                <td>
```
This doesn't do anything (same for all the other `<td>`s

Reply:
Fixed all instances.